services/user.py

```diff
@@ -0,0 +1,31 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                **kwargs) -> None:
+    get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        **kwargs
+    )
+
+
+def get_user(user_id: int) -> User:
+    return get_user_model().objects.get(id=user_id)
+
+
+def update_user(user_id: int,
+                password: str = None,
+                **kwargs) -> None:
+    user = get_user(user_id)
+
+    if password:
+        user.set_password(password)
+
+    for data_field, data in kwargs.items():
+        setattr(user, data_field, data)
```

Comment:
fields = ["email", ...]
You need to make additional check 'if data and data_field in fields' to check that your data is not None or your data is not "" and to check that your function shouldn't try to create object if somebody write 'udpate_user(1, "qwertyui", my_child_dream='popularity')' because model User/AbstractUser has no column 'my_child_dream' and it causes User.DoesNotExist error